apps/crn-server/src/controllers/research-outputs.ts

```diff
@@ -187,12 +223,18 @@ export default class ResearchOutputs implements ResearchOutputController {
   }
 }
 
+type ResearchOutputFilter =
```

Comment:
I guess that in the future we should refactor this so that only this new "object" filter will stay in the code.

Reply:
yeah i guess? cause this string filter makes no sense to me, like i dont know who thought that just 'filter' would be a name thats good enough for something that only refers to filtering types